tests/FSharp.Compiler.ComponentTests/EmittedIL/StringFormatAndInterpolation.fs

```diff
@@ -34,3 +34,22 @@ IL_0017:  ret"""]
 
 #endif
 
+    [<Fact>]
+    let ``Interpolated string with 3 parts consisting only of strings is lowered to concat`` () =
```

Comment:
I think there are more testing opportunities here:
- 1, 2, and 4 parts
- negative cases for more parts
- literals and functions also?

Reply:
Perhaps. Negative case with more parts doesn't really add much value for IL tests - there is a test elsewhere making sure it provides expected results in terms of behavior. Benchmarks kinda confirm that the optimization is applied in those other cases now, but I guess I cannot rule out a weird corner case regression in the future.

Reply:
Yeah well primarily I just thought those would be easy to add

Reply:
Added some 👍